src/facade/reply_builder.cc

```diff
@@ -120,6 +120,27 @@ void SinkReplyBuilder::StopAggregate() {
 MCReplyBuilder::MCReplyBuilder(::io::Sink* sink) : SinkReplyBuilder(sink), noreply_(false) {
 }
 
+void MCReplyBuilder::SendCommonString(CommonStrings string) {
+  if (noreply_)
+    return;
+
+  iovec v[1];
+
+#define X(e)                 \
+  case CommonStrings::e:     \
+    v[0] = IoVec(#e "\r\n"); \
+    break;
+
+  switch (string) {
+    DFLY_COMMON_STRINGS_X_MACRO()
```

Comment:
I find unfolding this macro call to be unreadable. Are we only trying to avoid the concatenation of `\r\n` in runtime?
If so, how about in the `.h` to just expose:

```cpp
enum CommonStrings {
  STORED,
  END,
  ...
  COMMON_STRINGS_SIZE,
};
```

And then in some `.cc` file do something like:

```cpp
std::array<string_view, CommonStrings::COMMON_STRINGS_SIZE> array;
```
And fill that array with some local-macro magic?